admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js

```diff
@@ -92,4 +92,42 @@ export default class OrderViewPage {
       this.listenForProductEdit();
     });
   }
+
+  listenForProductPagination() {
+    $(OrderViewPageMap.productsTablePagination).on('click', '.page-link', (event) => {
+      const $btn = $(event.currentTarget);
```

Comment:
It could be nice to add a `event.preventDefault();` here, since the clicked link has a '#' as a href value. It will keep the link from pointing to the top of the page, which can be confusing if the user has scrolled down even a little down...   (same for other click events on links I guess ?)

Reply:
:ballot_box_with_check: Thanks